styles/component-styles/text-utilities.css

```diff
@@ -0,0 +1,40 @@
+.text-utilities-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 2rem;
+    background-color: var(--secondary-container-color);
+    flex-wrap: wrap; 
+}
+
+.xs{
+    font-size: 0.8rem;
+}
+
+.s{
+    font-size: 1rem;
+}
+
+.m{
+    font-size: 1.5rem;
+}
+
+.l{
+    font-size: 2rem;
+}
+
+.xl{
+    font-size: 2.5rem;
+}
+
+.gray-text{
+    color: var(-gray-text-color);
```

Comment:
the variable name is missing an extra hyphen (-) in the beginning

Reply:
Thanks for pointing it out